jquery.kinetic.js

```diff
@@ -226,6 +224,12 @@
         if (self._useTarget(e.target, e) && self.elementFocused){
           return false;
         }
+      },
+      // prevent selection when dragging
+      selectStart: function (e){
+        if (self._useTarget(e.target, e)){
```

Comment:
It's fine moving the selectStart handler but it doesn't make sense to have this conditional. The point of this function is to stop text selection while dragging so even if it's ok to use the target for dragging the window it's probably not ok to start selecting or highlighting text